keyboards/POS78/README.md

```diff
@@ -0,0 +1,16 @@
+# POS78
+
+* Keyboard Maintainer: [smssmssms](https://github.com/smssmssms)
+* Hardware Supported: ATMEGA 32U4 (Adafruit Itsy Bitsy 5v) 
+* Hardware Availability: 1980-90's Partner Tech Corp KB-78 78 key Point-Of-Sale ortholinear keyboard conversion
+  https://www.partner.com.tw/wp-content/uploads/2019/09/KB-78-G-78-1-190911.pdf
+
+Make example for this keyboard (after setting up your build environment):
+
+    make POS78:default
+
+Flashing example for this keyboard:
+
+    make POS78:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
```suggestion
Example on how to compile and flash the keyboard (after setting up the build environment](https://docs.qmk.fm/#/getting_started_build_tools))

    qmk flash -kb pos78 -km default

Only want to compile the firmware?

    qmk compile -kb pos78 -km default
   
Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Since the qmk has moved towards mainly using the CLI for compiling/flashing I think it'd make sense to recommend that in the readme as well :)